games/devtest/mods/unittests/player.lua

```diff
@@ -4,33 +4,40 @@
 local expect = nil
 minetest.register_on_player_hpchange(function(player, hp, reason)
 	if expect == nil then
-		return
+		return hp
 	end
 
 	for key, value in pairs(reason) do
 		assert(expect[key] == value)
 	end
-	for key, value in pairs(expect) do
-		assert(reason[key] == value)
-	end
+    for key, value in pairs(expect) do
```

Comment:
It appears your editor is misconfigured to use spaces instead of tabs (for Lua)? Our code style guidelines mandate tabs, both for Lua and C++ code.